<launch>
    <node pkg="pid_car_control" type="control_jetson.py" name="jetson_controller" output="screen">
        <param name="interface" value="can0" />
        <param name="baudrate" value="250000" />
        <param name="odrive_dbc" value="$(find can_node)/odrive.dbc" />

        <remap from="/input/control_command" to="/control/control_command" />
    </node>
</launch>